### DISCUSSION TITLE
Master Clue - Bind Blood Rune at the Blood Altar - Incorrect info

### DISCUSSION THREAD

**Discussion by Nowimeanu** [runelite/runelite#16886] [Ideas]
The clue helper says that "dark essence fragments" are required.
I thought it was strange, but obliged. However, i do not have 77 rc yet, therefore i require boosting.
Get my 77 boost, and dark essence fragments do not work. 
Bad info.

**Comment by Felanbird** [comment]
I did this step twice yesterday and can confirm using dark essence fragments work as expected, but I have the rc requirement and do not require boosting.
